python/tvm/contrib/pipeline_executor.py

```diff
@@ -199,12 +290,43 @@ def is_pipeline_executor_interface(self):
             return not isinstance(self.io_owner, PipelineConfig.ModuleWrapper)
 
         def __repr__(self):
-            # Get all binding information.
-            ret = "  |{}: ".format(self.name)
+            # Get the binding information in the form of string.
+            str_format = "  |{}: ".format(self.name)
             for binding in self.bindings:
                 mname, dname = binding.get_name()
-                ret += "{0}:{1} ".format(mname, dname)
-            return ret
+                str_format += "{0}:{1} ".format(mname, dname)
+
+            return str_format
+
+        def check_dict(self, connection_dict):
```

Comment:
`dict` is too general and has no information. Better to be `check_binding_dict`.

Reply:
fixed.